joblib/numpy_pickle_utils.py

```diff
@@ -48,29 +39,18 @@
 except ImportError:
     bz2 = None
 
+# Register supported compressors
+register_compressor('zlib', CompressorWrapper(obj=BinaryZlibFile,
+                                              prefix=_ZLIB_PREFIX,
+                                              extension='.z'))
+register_compressor('gzip', CompressorWrapper(obj=BinaryGzipFile,
+                                              prefix=_GZIP_PREFIX,
+                                              extension='.gz'))
+register_compressor('bz2', BZ2CompressorWrapper())
+register_compressor('lzma', LZMACompressorWrapper())
+register_compressor('xz', XZCompressorWrapper())
+register_compressor('lz4', LZ4CompressorWrapper())
```

Comment:
Yes, I wanted to keep this at the very beginning of the module file. If I move this in compressor.py, the compressors will have to be registered at the very end of the file.

Maybe moving the register_compressor calls to numpy_pickle.py would be better. What do you think ?

Reply:
Yes I think this would make sense to move `register_compressor` in `numpy_pickle.py` as `_COMPRESSOR` is never used in the `compressor.py`. This would clarify the fact that the registration is done for the `dump` method of `numpy_pickle.py`.

Reply:
Yes I think this would make sense to move `register_compressor` in `numpy_pickle.py` as `_COMPRESSOR` is never used in the `compressor.py`. This would clarify the fact that the registration is done for the `dump` method of `numpy_pickle.py`.

Reply:
Ok, I moved them to numpy_pickle.py